server/graphql/v2/interface/IsMemberOf.js

```diff
@@ -48,8 +49,7 @@ export const IsMemberOfFields = {
         description: 'Filters on the Host fees structure applied to this account',
       },
       orderBy: {
-        type: new GraphQLNonNull(ChronologicalOrderInput),
-        defaultValue: { field: 'createdAt', direction: 'ASC' },
+        type: ChronologicalOrderInput,
```

Comment:
It's good practice to force having a predictable order. I don't get the reason for this change?

Reply:
I implemented the default ordering in the resolver itself.
The reason behind this is that the default ordering we had before (number of contributors and amount contributed) is much more complex.